content/concepts/content-management/031-fields.md

```diff
@@ -0,0 +1,103 @@
+
+## Field definition
+
+As fields build up content, the set of contained fields are defined when content types are defined. The content type definition (CTD) for different types holds thefield definition information besides a couple of type-related configuration settings. When defining a content type the contained fields can be defined in the CTD with XML fragments describing the type of the field, metadata and configuration:
+
+```xml
+<Field name="MyField" type="ShortText">
+  <DisplayName>My field</DisplayName>
+  <Description>Field to contain custom data</Description>
+  <Bind property="AnotherField" />
+  <Indexing>
+    <Mode>Analyzed</Mode>
+    <Store>Yes</Store>
+    <Analyzer>Keyword</Analyzer>
+  </Indexing>
+  <Configuration>
+    <ReadOnly>false</ReadOnly>
+    <Compulsory>true</Compulsory>
+    <OutputMethod>Raw</OutputMethod>
+    <DefaultValue>lorem ipsum</DefaultValue>
+    <VisibleBrowse>Hide</VisibleBrowse>
+    <VisibleEdit>Show</VisibleEdit>
+    <VisibleNew>Hide</VisibleNew>
+    <FieldIndex>101</FieldIndex>
+    <ControlHint>sn:CustomControl</ControlHint>
+  </Configuration>
+</Field>
+```
+&nbsp;
+&nbsp;
+
+| | |
+|-|-|
+| name | Name of the field. Required, should be unique in a content type. |
+| type | Short name of the field type. This or handler is required. (see the list of field types above) |
+| handler | Fully qualified type name of field handler class. This or handler is required. |
+| DisplayName | Displayed name of the field. |
+| Description | Description of the field. |
+| Bind | Name of the storage property the field is bound to. |
+| Indexing | Indexing settings of the field. |
+| ReadOnly | Indicates if the field is read-only. |
+| Compulsory | Indicates if the field is required to fill. |
+| OutputMethod | Defines the field output type (Default, Raw, Text, Html) |
+| DefaultValue | Default value of the field. |
+| VisibleBrowse | Indicates if the field is visible in autogenerated browse view. |
+| VisibleEdit | Indicates if the field is visible in autogenerated edit view (while a content is edited). |
+| VisibleNew | Indicates if the field is visible in autogenerated new view (while a content is created). |
+| FieldIndex | Defines order of field in autogenerated views. |
+| ControlHint | Default field control name. Used by the mapper in control libs like [sn-controls-react](https://github.com/SenseNet/sn-client/tree/develop/packages/sn-controls-react). |
+
+## Search in fields
+
+For every content the field values can be indexed so that when searched for a value the corresponding content will appear in the result set. It is also possible to search in fields by explicitely defining the them. The way a specific field of a content is indexed is defined in the [field definition](/concepts/basics/04-search#metadataindexing,searchingbyfields).
```

Comment:
```suggestion
For every content the field values can be indexed so that when searched for a value the corresponding content will appear in the result set. It is also possible to search in fields by explicitly defining them. The way a specific field of a content is indexed is defined in the [field definition](/concepts/basics/04-search#metadataindexing,searchingbyfields).
```